crypto/cipher/aes_icm.c

```diff
@@ -92,7 +92,7 @@ srtp_debug_module_t srtp_mod_aes_icm = {
  *
  */
 
-static srtp_err_status_t srtp_aes_icm_alloc_ismacryp (srtp_cipher_t **c, int key_len, int forIsmacryp)
+static srtp_err_status_t srtp_aes_icm_alloc (srtp_cipher_t **c, int key_len)
```

Comment:
Btw - it is gcc 6.3.1 that complains about incompatible pointers... I am unsure if it will trigger any potential issues.